.changeset/cool-pianos-smell.md

```diff
@@ -0,0 +1,9 @@
+---
+'@astrojs/vercel': major
+---
+
+Turn off `functionPerRoute` by default
+
+In the previous version of `@astrojs/vercel`, the default for `functionPerRoute` was changed to `true`. While this option has several advantages, if you a free tier user you are likely to run into the limit of 12 functions per deployment. This will result in an error when you attempt to deploy.
```

Comment:
```suggestion
In the previous version of `@astrojs/vercel`, the default for `functionPerRoute` was changed to `true`. While this option has several advantages, if you're a free tier user you are likely to run into the limit of 12 functions per deployment. This will result in an error when you attempt to deploy.
```